customers/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+class Customer(models.Model):
+    name = models.CharField(max_length=100)
+    email = models.EmailField()
+    phone = models.CharField(max_length=20)
+    address = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+class Interaction(models.Model):
+    customer = models.ForeignKey(Customer, on_delete=models.CASCADE, related_name='interactions')

```

Comment:
Please install the **Flake8** extension in your VS Code. After that, some warnings will help you with the styling. For example - this line is too long.